### PR TITLE
November Multilingual Publishing Comm Call - English

### PR DESCRIPTION
**Pull request by yabellini**
Creating placeholder

### CODE REVIEW

**Comment by yabellini** [comment]
@paocorrales and @eliocamp can you please, add your bios here :-) Thanks!!!

File: content/commcalls/2023-10-31/index.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  - url: /champions/
+    name:   Champions Program
+    details: web page with details about the champion program.
+speakers:  
+  - name: 
+    bio: 
+    img: /img/team/beatriz-milz-mentor.jpeg
+  - name: 
+    bio:         
+    img: /img/team/victor-ordu-mentee.jpg
```
Comment (eliocamp) [inline content/commcalls/2023-10-31/index.md:38]:
```suggestion
  - name: Elio Campitelli
    bio: "Elio is a a PhD student in atmospheric sciences at the Centre for Ocean and Atmospheric Research. They also maintain several open-source R packages (e.g., ggnewscale; metR) and contribute to other packages, such as data.table and ggplot2. Follow them on [mastodon](https://mastodon.social/@eliocamp) and check their [website](https://eliocamp.github.io/)."
    img: /img/team/elio-campitelli.jpg
```

File: content/commcalls/2023-10-31/index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+---
+title: "R in Goverment"
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:2]:
```suggestion
title: "Multilingual Publishing"
```

File: content/commcalls/2023-10-31/index.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  - url: 
+    name:   
+    details:  
+  - url: 
+    name: 
+  - url: /champions/
+    name:   Champions Program
+    details: web page with details about the champion program.
+speakers:  
+  - name:
```
Comment (paocorrales) [inline content/commcalls/2023-10-31/index.md:33]:
```suggestion
  - name: Pao Corrales
```

File: content/commcalls/2023-10-31/index.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    name:   
+    details:  
+  - url: 
+    name: 
+  - url: /champions/
+    name:   Champions Program
+    details: web page with details about the champion program.
+speakers:  
+  - name: 
+    bio:
```
Comment (paocorrales) [inline content/commcalls/2023-10-31/index.md:34]:
```suggestion
    bio: Pao is a PhD candidate in atmospheric sciences working on severe weather forecasts in Argentina. She is also a professor at Universidad Nacional Guillermo Brown teaching R and related tools. She also develops open-licensed teaching materials and contributes to several communities of practise related to R.
```

File: content/commcalls/2023-10-31/index.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    details:  
+  - url: 
+    name: 
+  - url: /champions/
+    name:   Champions Program
+    details: web page with details about the champion program.
+speakers:  
+  - name: 
+    bio: 
+    img: /img/team/beatriz-milz-mentor.jpeg
```
Comment (paocorrales) [inline content/commcalls/2023-10-31/index.md:35]:
```suggestion
    img: /img/team/paola-corrales.jpeg
```

**Review by paocorrales — COMMENTED** [review]
Adding my profile :)

File: content/commcalls/2023-10-31/index.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
+---
+title: "Multilingual Publishing"
+dateStart: 2023-10-21T12:00:00 # UTC!!
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:3]:
```suggestion
dateStart: 2023-11-23T12:00:00 # UTC!!
```

File: content/commcalls/2023-10-31/index.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+dateEnd: 2023-10-21T13:00:00 # UTC!!
+date: 2023-10-21T13:00:00 # UTC!! same as dateEnd
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:5]:
```suggestion
date: 2023-11-23T13:00:00 # UTC!! same as dateEnd
```

File: content/commcalls/2023-10-31/index.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+publishDate: 2023-08-19
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:6]:
```suggestion
publishDate: 2023-08-03
```

File: content/commcalls/2023-10-31/index.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+    bio: "Maëlle Salmon is a R(esearch) Software Engineer, part-time with rOpenSci where she, among other things, created and maintains the babeldown and babelquarto R packages, and maintains the guide [rOpenSci Packages: Development, Maintenance, and Peer Review](https://devguide.ropensci.org/). She also created the [R-hub blog](https://blog.r-hub.io) and co-wrote the book [HTTP testing in R](https://books.ropensci.org/http-testing) with [Scott Chamberlain](/author/scott-chamberlain). She lives in Nancy, France. She's an enthusiastic polyglot (of Latin and Germanic languages). Maëlle on [GitHub](https://github.com/maelle), [Mastodon](https://mastodon.social/@maelle), [Website](https://masalmon.eu/), [rOpenSci](/author/maëlle-salmon/)."
+    img: /img/team/maelle_salmon.jpg
+tags:
+  - community call
+  - events
+  - multilingual
+deets: |
+    Meeting ID:  875-494-77466
+    Passcode: 516675
+zoomurl: https://numfocus-org.zoom.us/j/87549477466?pwd=b0cydVpIYW5yMm9MKzZONG16RTVsUT09
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:49]:
```suggestion
zoomurl: https://numfocus-org.zoom.us/j/83866783222?pwd=Y1k3OEJnakc3WVRISlZxQ3NTUU1yZz09
```

File: content/commcalls/2023-10-31/index.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+    img: /img/team/elio-campitelli.jpg
+  - name: Maëlle Salmon
+    bio: "Maëlle Salmon is a R(esearch) Software Engineer, part-time with rOpenSci where she, among other things, created and maintains the babeldown and babelquarto R packages, and maintains the guide [rOpenSci Packages: Development, Maintenance, and Peer Review](https://devguide.ropensci.org/). She also created the [R-hub blog](https://blog.r-hub.io) and co-wrote the book [HTTP testing in R](https://books.ropensci.org/http-testing) with [Scott Chamberlain](/author/scott-chamberlain). She lives in Nancy, France. She's an enthusiastic polyglot (of Latin and Germanic languages). Maëlle on [GitHub](https://github.com/maelle), [Mastodon](https://mastodon.social/@maelle), [Website](https://masalmon.eu/), [rOpenSci](/author/maëlle-salmon/)."
+    img: /img/team/maelle_salmon.jpg
+tags:
+  - community call
+  - events
+  - multilingual
+deets: |
+    Meeting ID:  875-494-77466
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:47]:
```suggestion
    Meeting ID:  838-6678-3222
```

File: content/commcalls/2023-10-31/index.md [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+- Calendar 
+# vimeo:
+# - id: 
+resources:
+  - url: 
+    name:   
+    details:  
+  - url: 
+    name: 
+  - url: /champions/
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:29]:
```suggestion
  - url: /multilingual-publishing/
```

File: content/commcalls/2023-10-31/index.md [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+    name:   Champions Program
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:30]:
```suggestion
    name:   Multilingual Publishing
```

File: content/commcalls/2023-10-31/index.md [hunk 13]
```diff
@@ -0,0 +1,56 @@
...
+    details: web page with details about the champion program.
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:31]:
```suggestion
    details: web page with details about the Multilingual Publishing project.
```

File: content/commcalls/2023-10-31/index.md [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+attendees: All are welcome
+slug: nov2023-multilingual
+notes: 
+outputs:
+- HTML
+- Calendar 
+# vimeo:
+# - id: 
+resources:
+  - url:
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:24]:
```suggestion
  - url: https://devdevguide.netlify.app/es/preface.es.html
```

File: content/commcalls/2023-10-31/index.md [hunk 15]
```diff
@@ -0,0 +1,56 @@
...
+slug: nov2023-multilingual
+notes: 
+outputs:
+- HTML
+- Calendar 
+# vimeo:
+# - id: 
+resources:
+  - url: 
+    name:
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:25]:
```suggestion
    name:   rOpenSci DEvelopment Guide in Spanish
```

File: content/commcalls/2023-10-31/index.md [hunk 16]
```diff
@@ -0,0 +1,56 @@
...
+notes: 
+outputs:
+- HTML
+- Calendar 
+# vimeo:
+# - id: 
+resources:
+  - url: 
+    name:   
+    details:
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:26]:
```suggestion
    details:  First version of the rOpenSci Packages: Development, Maintenance, and Peer Review in Spanish
```

**Review by yabellini — COMMENTED** [review]
Adding resources and zoom information

File: content/commcalls/2023-10-31/index.md [hunk 17]
```diff
@@ -0,0 +1,56 @@
...
+outputs:
+- HTML
+- Calendar 
+# vimeo:
+# - id: 
+resources:
+  - url: https://devdevguide.netlify.app/es/preface.es.html
+    name:   rOpenSci DEvelopment Guide in Spanish
+    details:  First version of the rOpenSci Packages: Development, Maintenance, and Peer Review in Spanish
+  - url:
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:27]:
```suggestion
  - url: https://docs.ropensci.org/babeldown/
```

File: content/commcalls/2023-10-31/index.md [hunk 18]
```diff
@@ -0,0 +1,56 @@
...
+- HTML
+- Calendar 
+# vimeo:
+# - id: 
+resources:
+  - url: https://devdevguide.netlify.app/es/preface.es.html
+    name:   rOpenSci DEvelopment Guide in Spanish
+    details:  First version of the rOpenSci Packages: Development, Maintenance, and Peer Review in Spanish
+  - url: 
+    name:
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:28]:
```suggestion
    name: babeldown R package
    details: Translate Markdown-based content via DeepL API
```

File: content/commcalls/2023-10-31/index.md [hunk 19]
```diff
@@ -0,0 +1,57 @@
...
+# vimeo:
+# - id: 
+resources:
+  - url: https://devdevguide.netlify.app/es/preface.es.html
+    name:   rOpenSci DEvelopment Guide in Spanish
+    details:  First version of the rOpenSci Packages: Development, Maintenance, and Peer Review in Spanish
+  - url: https://docs.ropensci.org/babeldown/
+    name: babeldown R package
+    details: Translate Markdown-based content via DeepL API
+  - url: /multilingual-publishing/
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:30]:
```suggestion
  - url: https://docs.ropensci.org/babelquarto/
    name: babelquarto R package
    details: Set up and render a multilingual Quarto book
  - url: /multilingual-publishing/
```

File: content/commcalls/2023-10-31/index.md [hunk 20]
```diff
@@ -0,0 +1,60 @@
...
+slug: nov2023-multilingual
+notes: 
+outputs:
+- HTML
+- Calendar 
+# vimeo:
+# - id: 
+resources:
+  - url: https://devdevguide.netlify.app/es/preface.es.html
+    name:   rOpenSci DEvelopment Guide in Spanish
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:25]:
```suggestion
    name: rOpenSci DEvelopment Guide in Spanish
```

File: content/commcalls/2023-10-31/index.md [hunk 21]
```diff
@@ -0,0 +1,59 @@
...
+tags:
+  - community call
+  - events
+  - multilingual
+deets: |
+    Meeting ID:  838-6678-3222
+    Passcode: 584914
+zoomurl: https://numfocus-org.zoom.us/j/83866783222?pwd=Y1k3OEJnakc3WVRISlZxQ3NTUU1yZz09
+---
+
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:55]:
```suggestion
Open Source and Open Science, as global movements, faces language-based exclusion as most resources are in English. This affects scientists and research software engineers (RSE) working in R, particularly those who don't have English as first language. 

rOpenSci multilingual efforts aim to lower access barriers, democratize quality resources, and increase the possibilities of contributing to open software and science. We successfully piloted our Spanish-language peer review and the localization to Spanish of our comprehensive guide to software development, with Portuguese translation underway.
```

File: content/commcalls/2023-10-31/index.md [hunk 22]
```diff
@@ -0,0 +1,61 @@
...
+  - multilingual
+deets: |
+    Meeting ID:  838-6678-3222
+    Passcode: 584914
+zoomurl: https://numfocus-org.zoom.us/j/83866783222?pwd=Y1k3OEJnakc3WVRISlZxQ3NTUU1yZz09
+---
+Open Source and Open Science, as global movements, faces language-based exclusion as most resources are in English. This affects scientists and research software engineers (RSE) working in R, particularly those who don't have English as first language. 
+
+rOpenSci multilingual efforts aim to lower access barriers, democratize quality resources, and increase the possibilities of contributing to open software and science. We successfully piloted our Spanish-language peer review and the localization to Spanish of our comprehensive guide to software development, with Portuguese translation underway.
+On this call _Maëlle_, _Paola_ and _Elio_ will share details of the rOpenSci multilingual project. Maëlle will present the R packages that allow us to have our content in several languages while Elio and Paola will share how is the translation process and present the Translation Guide that was written to document this process.
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:58]:
```suggestion

On this call _Maëlle_, _Paola_ and _Elio_ will share details of the rOpenSci Multilingual project. Maëlle will present the R packages that allow us to have our content in several languages while Elio and Paola will share how is the translation workflow and present the Translation Guide that was written to document the process. 
```

**Pull request merged**